default/stringtables/en.txt

```diff
@@ -5473,6 +5499,32 @@ Repairs are delayed for one turn for ships just entering the system. Ships need
 
 Any combat in the system will disrupt repairs for that turn.'''
 
+CONFIG_GUIDE_TITLE
+Configuration Guide
+
+CONFIG_GUIDE_TEXT
+'''Most configuration options for the game are stored and read from a single XML formatted file named config.xml.
+If this file does not exist (or is for a different build or version) it will be (re)created when the client starts. 
+When an option is changed, the configuration file is updated.
```

Comment:
Not necessarily when changed, I think? Possibly sometimes only when "Done" is clicked, though I suspect it is inconsistent. Could you double-check?

Reply:
You are correct, options in the options wnd are not written until Done is clicked (or some other action occurs, such as clicking Apply on video tab).  I've changed the wording as there may be other areas where the option is not written to file until a later commit.